interop-testing/src/main/java/io/grpc/testing/integration/AbstractTransportTest.java

```diff
@@ -811,6 +811,17 @@ public AccessToken refreshAccessToken() throws IOException {
         authScope.contains(response.getOauthScope()));
   }
 
+  /** Sends an rpc to an unimplemented method on the server. **/
+  public void unimplementedMethod() {
+    UnimplementedServiceGrpc.UnimplementedServiceBlockingStub stub =
+        UnimplementedServiceGrpc.newBlockingStub(channel);
+    try {
+      stub.unimplementedCall(Empty.getDefaultInstance());
+    } catch (Throwable t) {
```

Comment:
Catching Throwable will swallow the AssertionError that fail() throws.  Catch Exception instead.